models/build.gradle.kts

```diff
@@ -56,14 +52,23 @@ kotlin {
             }
         }
     }
-    js() {
+    @OptIn(ExperimentalWasmDsl::class) wasmWasi {
+        nodejs()
+        binaries.executable()
```

Comment:
this should be library, same for js.
idk why i put these executable in the first CL, sorry.

Reply:
done